packages/medusa/src/services/order-edit.ts

```diff
@@ -382,4 +383,42 @@ export default class OrderEditService extends TransactionBaseService {
 
     return orderEdit
   }
+
+  async requestConfirmation(
+    orderEditId: string,
+    context: {
+      loggedInUser?: string
+    }
+  ): Promise<OrderEdit> {
+    return await this.atomicPhase_(async (manager) => {
+      const orderEditRepo = manager.getCustomRepository(
+        this.orderEditRepository_
+      )
+
+      let orderEdit = await this.retrieve(orderEditId, {
+        relations: ["changes"],
+        select: ["id", "requested_at", "changes"],
```

Comment:
nit(non-blocking): believe that fields specified in relations don't also have to be selected.

```suggestion
        select: ["id", "requested_at"],
```